api/types/types.go

```diff
@@ -229,13 +227,16 @@ type ContainerJSONBase struct {
 
 type ContainerJSON struct {
 	*ContainerJSONBase
-	Config *runconfig.Config
+	MountPoints map[string]MountPoint
+	Config      *runconfig.Config
 }
 
 // backcompatibility struct along with ContainerConfig
-type ContainerJSONRaw struct {
+type ContainerJSONPre120 struct {
 	*ContainerJSONBase
-	Config *ContainerConfig
+	Volumes   map[string]string
```

Comment:
we're not changing the old api. Those fields are in `ContainerJSONBase` and need to be present in the old api, that's why I moved them here.

`ContainerJSONBase` doesn't include info about volumes anymore. `ContainerJSONPre120` includes the info for the old api and `ContainerJSON` includes the info for the new api.